platform/zilliqa/api.go

```diff
@@ -1,23 +1,26 @@
 package zilliqa
 
 import (
+	"strconv"
+
 	"github.com/trustwallet/blockatlas/coin"
 	"github.com/trustwallet/blockatlas/pkg/blockatlas"
-	"strconv"
 
 	"github.com/spf13/viper"
 )
 
 type Platform struct {
 	client    Client
 	rpcClient RpcClient
+	udClient  blockatlas.Request
```

Comment:
It's better to extend, and you can use more simple

```suggestion
	blockatlas.Request
```

`err := p.udClient.Get(&resp, "/"+name, nil)`